signify-ts-test/COMMANDS.md

```diff
@@ -11,10 +11,12 @@ See the README.md in this dir for more information on how to run the tests and s
 
 # Example for how to run the verification test against a remote keria instance that has its own test data (like nord_demo)
 
-- Example command for using NordLEI dry-run identity to sign a report and then verify it against local
-`SIGNIFY_SECRETS="BhqEDDNmpyWgxT8ZIKWdw" TEST_ENVIRONMENT="nordlei_dry" ROLE_NAME="testbank_submitter" REG_PILOT_API=http://127.0.0.1:8000 VLEI_VERIFIER=http://127.0.0.1:7676 ./test.sh --build --report --verify`
+- Example command for using NordLEI demo identity to sign a report and then verify it against rootsid dev API/Verifier:
+  `SIGNIFY_SECRETS="A7DKYPya4oi6uDnvBmjjp" TEST_ENVIRONMENT="nordlei_demo" ROLE_NAME="unicredit-datasubmitter" REG_PILOT_API="https://reg-api-dev.rootsid.cloud" ./test.sh --build --report --verify`
```

Comment:
@bogtieba @tibor19 @dawgithub @pgovindeba thank you for the meeting today. @ronakseth96 updated the api/verifier and I successfully ran with the Nord KERIA ID and the deployed RootsID api/verifier with this command which I have added to the list of examples. Please try it an let us know how your testing is going.